File: Lecture 3/FinalCode - Java/app/src/main/java/com/inovaeg/androidtrainingpharos2020/examples/ManipulateViewsAtRuntimeActivity.java
package com.inovaeg.androidtrainingpharos2020.examples;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

import com.inovaeg.androidtrainingpharos2020.R;

public class ManipulateViewsAtRuntimeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_manipulate_views_at_runtime);

        TextView textView = findViewById(R.id.tv_manipulate_views_at_runtime);
        textView.setText("Manipulated!");
        textView.setBackgroundColor(getResources().getColor(R.color.colorAccent));
        textView.setPadding(50, 50, 50, 50);
    }
}

File: Lecture 3/FinalCode - Kotlin/app/src/main/java/com/inovaeg/androidtrainingpharos2020/examples/ManipulateViewsAtRuntimeActivity.kt
package com.inovaeg.androidtrainingpharos2020.examples

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.core.content.ContextCompat
import com.inovaeg.androidtrainingpharos2020.R
import kotlinx.android.synthetic.main.activity_manipulate_views_at_runtime.*

class ManipulateViewsAtRuntimeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_manipulate_views_at_runtime)
        tv_manipulate_views_at_runtime.apply {
            text = "Manipulated!"
            setBackgroundColor(ContextCompat.getColor(this@ManipulateViewsAtRuntimeActivity, R.color.colorAccent))
            setPadding(50, 50, 50, 50)
        }

    }
}